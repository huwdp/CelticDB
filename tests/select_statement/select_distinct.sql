CREATE TABLE test;
ALTER TABLE test ADD id INT;
ALTER TABLE test ADD text VARCHAR(255);
INSERT INTO test (id, text) VALUES (1, 'hello1');
INSERT INTO test (id, text) VALUES (1, 'hello1');
INSERT INTO test (id, text) VALUES (2, 'hello2');
INSERT INTO test (id, text) VALUES (2, 'hello2');
INSERT INTO test (id, text) VALUES (3, 'hello3');
SELECT DISTINCT * FROM test;
SELECT DISTINCT text, id FROM test;