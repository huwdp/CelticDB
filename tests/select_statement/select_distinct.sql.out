Parser: Create table named 'test'
Parser: Alter table named 'test'
Parser: Alter table named 'test'
Parser: Insert into table named 'test'
Parser: Insert into table named 'test'
Parser: Insert into table named 'test'
Parser: Insert into table named 'test'
Parser: Insert into table named 'test'
Parser: Select from table named 'test'
Parser: Select from table named 'test'
Command: Create statement
Command: Alter statement
Command: Alter statement
Command: Insert statement
Command: Insert statement
Command: Insert statement
Command: Insert statement
Command: Insert statement
Command: Select statement
Results:

 id              | text            |
 1               | 'hello1'        |
 2               | 'hello2'        |
 3               | 'hello3'        |

Command: Select statement
Results:

 text            | id              |
 'hello1'        | 1               |
 'hello2'        | 2               |
 'hello3'        | 3               |

